ccic/tiler.py

```diff
@@ -124,7 +149,7 @@ def get_slices(self, i, j):
         if j >= self.N - 1:
             j_clip_r = self.tile_size[1]
         else:
-            j_clip_r = self.tile_size[0] - self.j_clip[j]
+            j_clip_r = self.tile_size[1] - self.j_clip[j]
```

Comment:
I don't follow well this method, but this seemed like a typo. Correct? @simonpf

Reply:
Yes, that's a mistake.